io.itch.drwhut.TabletopClub.json

```diff
@@ -0,0 +1,57 @@
+{
+    "app-id": "io.itch.drwhut.TabletopClub",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "tabletop-club",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=xdg-desktop/TabletopClub:ro",
+        "--filesystem=xdg-documents/TabletopClub:create",
+        "--filesystem=xdg-download/TabletopClub:ro"
+    ],
+    "modules": [
+        {
+            "name": "tabletop-club",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D -m 755 -p TabletopClub.x86_64 /app/bin/tabletop-club",
+                "install -D -m 755 -p -t /app/lib/ libwebrtc_native.linux.release.64.so",
+                "install -D -m 644 -p -t /app/share/TabletopClub/ TabletopClub.pck",
+                "mv assets/ /app/share/TabletopClub/",
+                "install -D -m 644 -p tabletop_club_icon.png /app/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png",
+                "install -D -m 644 -p -t /app/share/metainfo/ $FLATPAK_ID.metainfo.xml",
+                "install -D -m 644 -p -t /app/share/applications/ $FLATPAK_ID.desktop"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/drwhut/tabletop-club/releases/download/v0.1.0/TabletopClub_v0.1.0_Linux_64.zip",
+                    "sha512": "bf1fee8b8b86941ea289a287db65cdeaca729c675c148d9b0c2a851a0ab0b2dddd7106a2975518c6cedb515b5d084e07f9e8f39a65e52ffbe29d84c99a2f4e51",
+                    "strip-components": 0
+                },
+                {
+                    "type": "file",
+                    "url": "https://raw.githubusercontent.com/drwhut/tabletop-club/v0.1.0/game/tabletop_club_icon.png",
+                    "sha512": "85176d840093ff9dc4782067c1f30ea5b512406a6ab7b7c76a0ee010fbb7c34771bffdfbe30665192ed685769883a67ef460116f20da0f1b56f270b6e5ab2e4e"
+                },
+                {
+                    "type": "file",
+                    "url": "https://raw.githubusercontent.com/drwhut/tabletop-club/v0.1.0/misc/flatpak/io.itch.drwhut.TabletopClub.metainfo.xml",
+                    "sha512": "c738849b274775ebf499926de6bac177c5185cd96ab7d90fa4ad0a4281f07a584181d0f95ea8bade5e49389bf9ded7fce3f3a153c27c65e2266d2fd6ef6750ee"
+                },
+                {
+                    "type": "file",
+                    "url": "https://raw.githubusercontent.com/drwhut/tabletop-club/v0.1.0/misc/flatpak/io.itch.drwhut.TabletopClub.desktop",
```

Comment:
reading the desktop file, I suspect that `flatpak run io.itch.drwhut.TabletopClub` will not work. You should make a wrapper script instead.